cli/command/plugin/create_test.go

```diff
@@ -29,7 +33,7 @@ func TestCreateErrors(t *testing.T) {
 		},
 		{
 			args:          []string{"plugin-foo", "nonexistent_context_dir"},
-			expectedError: "no such file or directory",
+			expectedError: noSuchFile,
```

Comment:
Ah, I recall some shenanigans like https://github.com/Microsoft/opengcs/blob/8cfbb6d8ae8203a80464ca6e2bd39102d6fd9deb/service/gcsutils/remotefs/utils.go#L93-L107 and https://github.com/Microsoft/opengcs/blob/8cfbb6d8ae8203a80464ca6e2bd39102d6fd9deb/service/gcsutils/remotefs/utils.go#L36-L50

Reply:
🤔

Reply:
I think there is an `os.IsNotFound()` or `os.IsFileNotFound` function that would be prettier

Reply:
Oh! you're right, was thinking about Linux CLI calling a Windows daemon (in which errors don't match), but that's not the case here